modules/extloads/src/ExtLoads.f90

```diff
@@ -205,7 +265,7 @@ subroutine Init_y(y, u, m, p, errStat, errMsg)
       if (ErrStat >= AbortErrLev) RETURN         
 
       call MeshCopy ( SrcMesh  = u%TowerMotion    &
-           , DestMesh = y%TowerLoadAD      &
+           , DestMesh = u%TowerLoadAD      &
```

Comment:
Good catch on this.  I'm not sure how that was incorrectly put in `y` for so long.